chia/full_node/block_store.py

```diff
@@ -147,9 +147,9 @@ async def get_full_block(self, header_hash: bytes32) -> Optional[FullBlock]:
         return None
 
     async def get_full_block_bytes(self, header_hash: bytes32) -> Optional[bytes]:
-        cached = self.block_cache.get(header_hash)
+        cached: Optional[FullBlock] = self.block_cache.get(header_hash)
         if cached is not None:
-            return cached
+            return bytes(cached)
```

Comment:
@Yostra this was wrong, but I think it was still working due to how streamable takes bytes as arguments

Reply:
@mariano54 this was a hack to avoid serialization or desirialization. I forgot to change so that return type is Union[bytes,FullBlock] 
Can we change this or is there something else that now uses this method ?